label_semantic_roles/README.en.md

```diff
@@ -92,10 +92,10 @@ Fig 4. Bidirectional LSTMs
 
 线性变换-> linear transformation
 输入层到隐层-> input-to-hidden
-正向处理输出序列->process sequence in forward direction
-反向处理上一层序列-> process sequence from previous layer in backward direction
+正向处理输出序列->process sequence in forwarding direction
```

Comment:
typically people use `forward direction ` in the literature. I have not see `forwarding direction` before